sshd/net_test.go

```diff
@@ -25,7 +25,7 @@ func TestServerInit(t *testing.T) {
 }
 
 func TestServeTerminals(t *testing.T) {
-	signer, err := NewRandomSigner(512)
+	signer, err := NewRandomSigner(1024)
```

Comment:
Curious why you ended up switching tests from 512 bits to 1024 bit signers? Wouldn't it just make tests slower? Is there a benefit?

Reply:
For quite a time, github actions was timing out tests without me being able to reproduce locally. After 89ee27d, they consistently timed out on my side too and I could trace it down to an error related to key size ("message too large for key size", I think).

Reply:
Ah fascinating, thanks for explaining!